curvefs/src/client/s3/disk_cache_manager.cpp

```diff
@@ -470,7 +475,9 @@ void DiskCacheManager::TrimCache() {
                         << "error is: " << errno;
                     continue;
                 }
-                DecDiskUsedBytes(statReadFile.st_size);
+                metric::CollectMetrics(&metric_->trim_, statReadFile.st_size,
+                                       butil::cpuwide_time_us() - start);
+                UpdateDiskUsedBytes(-statReadFile.st_size);
```

Comment:
If there are some readcache and remounted, is the value of diskused correct? does it decrease to negative?

Reply:
> If there are some readcache and remounted, is the value of diskused correct? does it decrease to negative?

No, it will read from the cache disk and count again.